src/com/mie/model/Application.java

```diff
@@ -1,19 +1,29 @@
 package com.mie.model;
-
+import java.sql.Timestamp;
```

Comment:
I made the new AppDeadline attribute a date that shows up like "2020-12-01", which I think should work with java.sql.Date. Timestamp is date + time